docs/DeveloperGuide.md

```diff
@@ -210,7 +272,8 @@ Thus the data is not protected by nature.
 
 ## Glossary
```

Comment:
tbh i thought this should be things like "utilities component" or smth

Reply:
From the website is actually just a term for us to have a common understanding of certain key words ? 

Glossary: A glossary serves to ensure that all stakeholders have a common understanding of the noteworthy terms, abbreviations, acronyms etc.